src/webGLWorker.js

```diff
@@ -500,6 +500,10 @@ function WebGLWorker() {
         removeRunDependency('gl-prefetch');
         break;
       }
+      case 'removeRunDependency': {
```

Comment:
hmm, could you instead send the `setPrefetched` message that is handled right above this? then it would also remove that run dep, and the other stuff it does is not harmful.

that seems cleaner to me, as logically that's what's happening, we are giving up on prefetch if there is no GL.